<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tune</class>
 <widget class="QMainWindow" name="Tune">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>270</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>575</width>
    <height>393</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tune Music</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>575</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="background_image">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>573</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="background_frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>575</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 100), stop:0.318182 rgba(0, 0, 0, 20))}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="album_cover">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(255,255,255,30);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="title">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>391</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;
color: black;
font:17px</string>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
      <widget class="QSlider" name="progress_slider">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>210</y>
         <width>341</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">

QSlider::groove:horizontal {
    height: 3px; 
    background-color: rgba(255, 255, 255, 150);
    border-radius: 1px;
    padding-left: 0px;
    padding-right: 0px;
}

QSlider::handle:horizontal {
    background-color: #60cdff;
    border: 4px solid #454545;
    width: 9px;
    min-height: 13px;
    margin: -7px 0;
    border-radius: 8px;
}

QSlider::handle:horizontal:hover {
    background-color: #60cdff;
    border: 3px solid #454545;
    width: 11px;
    min-height: 13px;
    margin: -7px 0;
    border-radius: 8px;
}

QSlider::handle:horizontal:pressed {
    background-color: #60cdff;
    border: 5px solid #454545;
    width: 7px;
    min-height: 13px;
    margin: -7px 0;
    border-radius: 8px;
}

QSlider::groove:horizontal:disabled {
    background-color: rgba(255, 255, 255, 75);
}

QSlider::handle:horizontal:disabled {
    background-color: #454545;
    border: 6px solid #353535;
}</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="time">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>220</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;
color: black;</string>
       </property>
       <property name="text">
        <string>00:00:00/00:00:00</string>
       </property>
      </widget>
      <widget class="QLabel" name="author">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;
color: black;</string>
       </property>
       <property name="text">
        <string>Author</string>
       </property>
      </widget>
      <widget class="QLabel" name="node">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;
color: black;</string>
       </property>
       <property name="text">
        <string>Node (ping)</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>575</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tune Connect</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="links">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color:#2D2D2D;
    border-radius: 5px;
    color:white;
    font: 13px 'Segoe UI';
    color: #ffffff;
    border:1px solid #313131;
    padding: 7px 20px; 
    margin-right: 5px;
}

QPushButton:hover {
    background-color:#343434
;
}

QPushButton:pressed,
QPushButton:pressed:focus {
    background-color:#262626;
    border: 1px solid rgba(255, 255, 255, 8);
    color: #c8c8c8;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
